README.md

```diff
@@ -15,3 +15,39 @@ This repository houses the open-source components of the Mattermost Integrations
 ## Get Involved
 
 - [Join the discussion on Integrations Marketplace](https://community.mattermost.com/core/channels/marketplace)
+
+## Developing
+
+### Environment Setup
+
+1. Install [Go](https://golang.org/doc/install)
+
+### Building
+
+Simply run the following:
+
+```
+# Make sure Go modules is turned on
+$ export GO111MODULE=on
```

Comment:
What do you think about just requiring to clone this outside of `$GOPATH`?

Reply:
Alternatively, would it make sense to codify this into a `Makefile` as most of our projects seem to end up adopting?

Reply:
Using the `Makefile` totally makes sense, especially since we're using it in other places.
Thinking about keeping this as is till we employ the `Makefile`, unless folks feel we should add it now.

thoughts @hanzei @lieut-data?

Reply:
+1 on the Makefile